src/build/validator-restart.md

```diff
@@ -70,7 +70,7 @@ init 0
 ```
 
 Watch the Explorer page for that Validator, and wait until it gets jailed.
-This could take as long as 35 minutes.
+This could take around 33 hours.
 When the Explorer page indicates that teh validaor has been jailed, start the server back up again.
```

Comment:
Small typo here that's worth amending at the same time

Reply:
ok sorted.. and found a couple more